.github/workflows/docker-publish.yaml

```diff
@@ -33,6 +33,8 @@ jobs:
           go-version: "1.20.6"
         id: go
 
+      - uses: bufbuild/buf-setup-action@v1.27.1
```

Comment:
Slightly odd that we install it differently on the GitHub workflow and on a local dev box.  Price of using GH actions, I guess.